miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppPermission.kt

```diff
@@ -0,0 +1,24 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import android.Manifest
+
+internal object MiniAppPermission {
+
+    internal object Code {
+        const val GEOLOCATION = 1001
+    }
+
+    internal object PermissionType {
+        const val GEOLOCATION = "miniapp.permission.geolocation"
+    }
+
+    internal fun getPermissionRequest(permissionType: String) = when (permissionType) {
+        PermissionType.GEOLOCATION -> arrayOf(Manifest.permission.ACCESS_FINE_LOCATION)
```

Comment:
Should this have both `ACCESS_FINE_LOCATION` & `ACCESS_COARSE_LOCATION`?

Reply:
When `ACCESS_FINE_LOCATION` is approved, the `ACCESS_COARSE_LOCATION` is implicity approved too.

Reply:
As I mentioned above, I think this should be moved to the Host App side so they have that control (`ACCESS_FINE_LOCATION` vs `ACCESS_COARSE_LOCATION`)